packages/website/docs/docs/editor.md

```diff
@@ -1,6 +1,6 @@
 # Customizing the Editor
 
-There are a number of options that you can pass to `useBlockNote()`, which you can use to customize the editor. You can find the full list of these below:
+There are a number of options (all optional) that you can pass to `useBlockNote()`, which you can use to customize the editor. You can find the full list of these below:
```

Comment:
I think it makes more sense to just do `BlocknoteEditorOptions = Partial<{...}>`